opc-ua-sdk/sdk-client/src/main/java/org/eclipse/milo/opcua/sdk/client/subscriptions/MonitoredItemTransferRequest.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2019 the Eclipse Milo Authors
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+package org.eclipse.milo.opcua.sdk.client.subscriptions;
+
+import org.eclipse.milo.opcua.stack.core.types.builtin.unsigned.UInteger;
+import org.eclipse.milo.opcua.stack.core.types.enumerated.MonitoringMode;
+import org.eclipse.milo.opcua.stack.core.types.structured.MonitoringParameters;
+import org.eclipse.milo.opcua.stack.core.types.structured.ReadValueId;
+
+public class MonitoredItemTransferRequest {
+    private final UInteger clientHandle;
+    private UInteger serverHandle;
+
+    private final ReadValueId itemToMonitor;
+    private final MonitoringMode monitoringMode;
+    private final MonitoringParameters requestedParameters;
+
+    public MonitoredItemTransferRequest() {
```

Comment:
Do you need these zero-arg constructors? I'd like to delete them if not.

Reply:
Not at all. I add them out of habit from my c# days where some serializers needed them but would give cryptic errors when they were not there.